simulation/conveyor/ConveyorsEnvironment.py

```diff
@@ -203,6 +211,12 @@ def _leaveConveyorEnd(self, conv_idx: int, bag_id: int) -> bool:
         up_type = node_type(up_node)
 
         if up_type == "sink":
+            if self._path_memory is not None:
+                reward = -100
```

Comment:
Put `100` in object params

Reply:
I don't sure if it worth to pull it from config yet, put it in constant in init